tests/rptest/clients/rpk.py

```diff
@@ -106,20 +112,52 @@ def wasm_gen(self, directory):
         cmd = [self._rpk_binary(), 'wasm', 'generate', directory]
         return self._execute(cmd)
 
-    def _run_api(self, cmd, stdin=None, timeout=30):
+    def _run_topic(self, cmd, stdin=None, timeout=30):
         cmd = [
-            self._rpk_binary(), "api", "--brokers",
+            self._rpk_binary(), "topic", "--brokers",
             self._redpanda.brokers(1)
         ] + cmd
         return self._execute(cmd, stdin=stdin, timeout=timeout)
 
+    def cluster_info(self, timeout=30):
```

Comment:
this should probably be an interface in `clients/kafka_client.py`

Reply:
I took a look and I think `cluster_info` is an rpk-specific thing, so I don't know if it's a good idea.

Reply:
you can add a new abstraction. like if cluster_info returns some info about brokers or whatever, then that's something pretty any client can do.

Reply:
How is it RPK specific? I get that other clients / tools don't have a `cluster_info` command, but they can all support it at the API level in ducktape and allow us to write generic tests that work across all clients that we are testing. You don't need to implement it for all clients, but add the interface and make the data structure generic.

Reply:
How is it RPK specific? I get that other clients / tools don't have a cluster_info command, but they can all support it at the API level in ducktape and allow us to write generic tests that work across all clients that we are testing. You don't need to implement it for all clients, but add the interface and make the data structure generic.

Reply:
It's specific in the sense that only `rpk` has a command of that sort. I'm always hesitant to change an interface based on a single implementation's details, since an interface should be an abstraction. But I can do it.